src/org/daisy/dotify/formatter/impl/obfl/ObflParserImpl.java

```diff
@@ -1588,7 +1591,9 @@ private void parseExternalReference(
             XMLEventIterator input
     ) throws XMLStreamException {
         Iterator<Attribute> it = event.asStartElement().getAttributes();
-        Map<QName, String> attrList = new HashMap<>();
+        if (externalReferenceObject == null) {
```

Comment:
The checking on `null` should be removed everywhere

Reply:
Why?

Reply:
Please see my remark on line 123, under 3.